plugins/org.eclipse.xtend.core/src/org/eclipse/xtend/core/macro/ConstantExpressionsInterpreter.xtend

```diff
@@ -94,6 +94,10 @@ class ConstantExpressionsInterpreter extends AbstractConstantExpressionsInterpre
 	 * That essentially includes static imports and the fields declared in the containing types
 	 */
 	protected def Map<String, JvmIdentifiableElement> findVisibleFeatures(XExpression expression) {
+		// don't build a scope for a storage loaded resource, since everything is computed and we will resolve the proxies
+		switch res : expression.eResource {
```

Comment:
This will break the interpreter for statically imported constant values, e.g. they can no longer be used as annotation values.